app/scripts/ingest_census_tract_geo.py

```diff
@@ -0,0 +1,103 @@
+import requests
+from route_rangers_api.models import Demographics
```

Comment:
just an FYI: 
this probably works, but I think this should be `app.route_rangers_api.models` b/c your version will import locally but ideally it imports from the module version of this package, i.e. when you run `poetry install` this whole repo becomes its own package and then you can call things and not worry about being in a certain filepath for them to work.